pkg/lib/dialogs.jsx

```diff
@@ -82,19 +82,55 @@
  *   }
  * }
  *
+ * If you need to wait on the result of a Dialog before you can await "Dialogs.show"
```

Comment:
This sentence reads funny because of the "before". Maybe just: If you need to wait on the result of a Dialog, you can await "Dialogs.show"?